src/store/models/taro.ts

```diff
@@ -7,7 +7,7 @@ import { BLOCKS_TIL_CONFIRMED } from 'utils/constants';
 import { RootModel } from './';
 
 //This is the minimum balance that a taro node must have access to in order to mint assets
-export const TARO_MIN_LND_BALANCE = 10000;
+export const TARO_MIN_LND_BALANCE = 15000;
```

Comment:
Why change this minimum amount? Did you run into an issue? Changing it is redefining the meaning of the constant. Instead of increasing it, just update mintAsset and sendAsset to deposit `TARO_MIN_LND_BALANCE * 2` sats. This is what is done when opening a channel.

Reply:
Done